src/main/java/com/google/devtools/build/lib/bazel/rules/java/java_stub_template.txt

```diff
@@ -241,6 +241,13 @@ fi
 # Set JAVABIN to the path to the JVM launcher.
 %javabin%
 
+if is_windows; then
+  # Cygwin will not accept any backslashes in JAVABIN when trying to execute it.
+  JAVABIN="$(cygpath --unix $JAVABIN)"
+  # Ensure we have .exe suffix, JARBIN resolution fails without it.
```

Comment:
```suggestion
  # Ensure we have .exe suffix, JAVABIN resolution fails without it.
```

Do you know why this is needed? The path to the java binary should be set correctly by the toolchain with https://github.com/bazelbuild/rules_java/pull/133. Do you still need this file with Bazel built from master?

Reply:
Yeah I did mean`JARBIN`, not `JAVABIN`.  You can see here [java_stub_template.txt#L361-L364](https://github.com/bazelbuild/bazel/blob/8c70d7b4ad5a9299f8e0cf7d2209fca0a29fd96f/src/main/java/com/google/devtools/build/lib/bazel/rules/java/java_stub_template.txt#L361-L364) that on windows `JARBIN` resolution requires the .exe suffix.